lib/rf/stylez.rb

```diff
@@ -11,6 +14,25 @@
 
 module Rf
   module Stylez
-    # Your code goes here...
+    RUBYGEMS_URL = URI('https://rubygems.org/api/v1/gems/rf-stylez.json').freeze
+
+    def self.check_for_updates
+      remote_version = JSON.parse(Net::HTTP.get(RUBYGEMS_URL))['version']
```

Comment:
How often will this run if you have rf-stylez configured to be used by your IDE? Also, a timeout would be nice in case rubygems hanging stops you from working?

Maybe having this as a rake task would be better?

Reply:
We should just disable this for CI, good point.

I don't think rake task solves the problem since you'd have to run it - you might as well blindly update your local `rf-stylez`. Maybe there's a better way to fix this issue?

Reply:
I dunno. The advantage of this is that it runs every time you load the code, the disadvantage is it runs every time you load the code...

Reply:
Yeah not super in favor of this if it's gonna be running all the time, I do style checking through the editor and this seems like it could be problematic? (Unless it doesn't work the way I think it does.)

Reply:
My understanding is it'll execute every time the file is loaded, so it depending on how your editor invokes the checking it could be ok or not - although it'll probably mean a HTTP request for every change you make in the files being checked?

Reply:
That runs on every load and if I understand things correctly, that's every time `rubocop` is run.

I guess it's not worth it - the problem it's trying to solve is that our local installs might be out of data vs latest rf-stylez. Now that I think about it a commit hook makes way more sense and that's what I'll explore